app/src/main/res/values-v29/ccx_styles.xml

```diff
@@ -0,0 +1,16 @@
+<resources xmlns:tools="http://schemas.android.com/tools">
+
+    <style name="CCX_Screen" parent="Theme.AppCompat.NoActionBar">
+        <item name="android:windowTranslucentStatus">false</item>
+        <item name="android:statusBarColor">@android:color/transparent</item>
+        <item name="android:navigationBarColor">
+            <!-- This color is obtained experimentally.
+            Figma value doesn't match the rendered background bottom color-->
+            #0D0E12
+        </item>
+        <item name="android:windowBackground">@drawable/ccx_main_background</item>
+        <item name="android:forceDarkAllowed">false</item>
```

Comment:
Could you please try adding the attribute to the main styles with `tools:targetApi` to disable warning, instead of copying them to v29?